### DISCUSSION TITLE
does kiali support mcp?

### DISCUSSION THREAD

**Discussion by rtttech** [kiali/kiali#3721] [Q&A]
Is k8s api server is a must?

**Comment by lucasponce** [comment]
I'm afraid I don't understand the question.

**Reply by lucasponce** [reply]
Kiali reads multiple signals and sources from: Prometheus Telemetry, Istio Configuration, Kubernetes Workloads and Services, Tracing.

And it puts it together to build high level visualization and management services for Istio Service Mesh.

https://kiali.io/documentation/latest/architecture/

So, out of the box, these sources don't work separated, only tracing is an optional source, but then a portion of the correlation features won't be available to the user.

**Reply by lucasponce** [reply]
If your "mcp" refers to multicluster support, it's a WIP where available features are documented here:

https://kiali.io/documentation/latest/features/#_advanced_mesh_deployment_and_multi_cluster_support

**Reply by lucasponce** [reply]
@rtttech I moved to a discussion section, hope you don't get too many github notifications.

I think your questions suits better in this format and allow folks to comment in a better manner than an issue.
